### ISSUE TITLE
TypeError: write() takes no keyword arguments

### ISSUE THREAD

**Issue by orf** [Delgan/loguru#91]
Hey, thanks for this awesome library!

I'm running into a bit of an issue though. I've got this simple code:
```
from loguru import logger
logger.add(sys.stdout, format="", colorize=True, backtrace=True, diagnose=True)
logger.info('test')
```

On Python 3.6 I get the following traceback:
```
INFO     | x:<module>:12 - test
--- Logging error in Loguru Handler #1 ---
Record was: {'elapsed': datetime.timedelta(0, 0, 614), 'exception': None, 'extra': {}, 'file': '__init__.py', 'function': '<module>', 'level': 'INFO', 'line': 12, 'message': 'test', 'module': '__init__', 'name': 'x', 'process': '13540', 'thread': '4502959552', 'time': datetime(2019, 4, 23, 17, 44, 8, 710585, tzinfo=datetime.timezone(datetime.timedelta(0, 3600), 'WEST'))}
Traceback (most recent call last):
  File "x/loguru/_handler.py", line 213, in emit
    self.writer(str_record)
  File "/x/loguru/_logger.py", line 645, in writer
    write(m)
  File "/x/loguru/_logger.py", line 636, in write
    return stream_write(m, **kwargs)
TypeError: write() takes no keyword arguments
--- End of logging error ---
```

After digging a bit `kwargs` here is equal to `{'diagnose': True}`. If I don't pass `diagnose=True` to `logger.add()` it works as expected, with no traceback.

**Comment by Delgan** [comment]
Hello @orf.

Actually, the README does not properly reflect the release you get while installing `loguru`. The Github `master` branch contains the latest developed features. The version which implements the `diagnose` option has not yet been published. This is why you get this error. 

Take a look at the [`stable`](https://loguru.readthedocs.io/en/stable/index.html) documentation, this should match the more recent version. 👍

**Comment by Delgan** [comment]
Fixed now that `0.3.0` is officially released, the new `diagnose` argument can be used in `.add()` without error. :smiley:
